### ISSUE TITLE
Display the  differennt brands making same medicine ingredients.

### ISSUE THREAD

**Issue by rajlakshmi18704** [ajaynegi45/upchaar-aayog#6]
#6

1. User will enter the  medicine name in the input box.

2. Process allows you to take a medicine name input from the user and retrieve the corresponding brands by leveraging the RxNorm API.

3.It  involves fetching data from specific endpoints, starting from getting the RxCUI, then the ingredient ID(s), and finally fetching the brands associated with those ingredients.

4.Display the details for the each corresponding  brand  making the medicine.

**Comment by ajaynegi33** [comment]
> #6
> 
> 1. User will enter the  medicine name in the input box.
> 2. Process allows you to take a medicine name input from the user and retrieve the corresponding brands by leveraging the RxNorm API.
> 
> 3.It involves fetching data from specific endpoints, starting from getting the RxCUI, then the ingredient ID(s), and finally fetching the brands associated with those ingredients.
> 
> 4.Display the details for the each corresponding brand making the medicine.



Hi @rajlakshmi18704,

Your idea seems interesting! Let me know if you'd like to work on this issue, and I can assign it to you.

**Comment by rajlakshmi18704** [comment]
yes, I would love to work on this issue under hacktoberfest. please assign it to me.

**Comment by ajaynegi33** [comment]
> yes, I would love to work on this issue under hacktoberfest. please assign it to me.

Hi @rajlakshmi18704,

Thank you for expressing your interest in working on the "`Display the differennt brands making same medicine ingredients.`" issue. I'm delighted to inform you that I have assigned this issue to you. Your willingness to contribute to our project is much appreciated. For this project, I’d prefer we stick to pure CSS and avoid any CSS frameworks like Tailwind to keep it lightweight.

Feel free to start working, and if you have any questions or need assistance during the process, please don't hesitate to reach out.
